apis/nucleus/src/components/ActionsToolbarMore.jsx

```diff
@@ -55,6 +55,11 @@ const More = React.forwardRef(
           ref={ref}
           open={show}
           anchorEl={alignTo.current}
+          getContentAnchorEl={null}
+          container={alignTo.current}
+          disablePortal
+          hideBackdrop
+          style={{ pointerEvents: 'none' }}
```

Comment:
Instead of `style` it could send it in as a property and set through the `styled(Popover)` call.